overrides.mk

```diff
@@ -4,7 +4,7 @@
 
 # DEFAULT values
 DEFAULT_BASEIMAGE="registry.access.redhat.com/ubi8/ubi-minimal"
-DEFAULT_DIGEST="sha256:3aa3f379a81013bd3264faa0af87d201cdaa5981050d78c567b48fdfd5b38bb8"
+DEFAULT_DIGEST="sha256:21504085e8d740e62b52573fe9a1a0d58a3e7dba589cac69734ad2fa81d66635"
```

Comment:
suggestion : Just for clarity, add a comment indicating the exact version? I know SHA defines it, but i noticed some other drivers do that for informative purposes.

Reply:
added comment